addons/web/static/src/scss/bootstrap_overridden_frontend.scss

```diff
@@ -93,6 +98,11 @@ $form-range-thumb-disabled-bg: $input-disabled-bg !default;
 $form-file-button-bg: mix($input-color, $input-bg, $btn-hover-bg-shade-amount * 0.3) !default;
 $form-file-button-hover-bg: mix($input-color, $input-bg, $btn-hover-bg-shade-amount) !default;
 
+// Dropdowns
```

Comment:
missing `web` and `web_editor` in the modules list of the commit title 
could you upadte it before I merge ? thanks